build/hiredis/Makefile

```diff
@@ -0,0 +1,84 @@
+
+ROOT=../..
+MK.pyver:=3
+include $(ROOT)/deps/readies/mk/main
+
+define HELPTEXT
+make build    # configure and compile
+make clean    # clean generated sbinaries
+  ALL=1       # remote entire binary directory
+endef
+
+MK_ALL_TARGETS=build
+
+#----------------------------------------------------------------------------------------------
+
+BINDIR=$(BINROOT)/hiredis
+SRCDIR=$(ROOT)/deps/LibMR/deps/hiredis
```

Comment:
@rafie should we implement this in LibMR and assume LibMR is doing the right think when we compile it? it feels a bit funky that we touch/compile the LibMR deps.